holoviews/plotting/bokeh/element.py

```diff
@@ -286,16 +295,139 @@ def _hover_opts(self, element):
         dims += element.dimensions()
         return list(util.unique_iterator(dims)), {}
 
+    def _replace_label_group(self, element, tooltip):
```

Comment:
All these methods, in my opinion, need to start with `_hover` to understand that they are easily related to the hover tool.